README.md

```diff
@@ -7,6 +7,29 @@ An opinionated static site generator written in Rust.
 Documentation is available on [its site](https://www.getgutenberg.io/documentation/getting-started/installation/) or
 in the `docs/content` folder of the repository.
 
+## Comparisons with other static site generators
+
+|                          | Gutenberg | Cobalt | Hugo | Pelican |
+|--------------------------|-----------|--------|------|---------|
+| Single binary            |     ✔     |    ✔   |   ✔  |    ✕    |
+| Language                 |    Rust   |  Rust  |  Go  |  Python |
+| Syntax highlighting      |     ✔     |    ✔   |   ✔  |    ✔    |
+| Sass compilation         |     ✔     |    ✔   |   ✕  |    ✔    |
+| Assets co-location       |     ✔     |    ✕   |   ✔  |    ✔    |
+| i18n                     |     ✕     |    ✕   |   ✔  |    ✔    |
+| Image processing         |     ✕     |    ✕   |   ✔  |    ✔    |
+| Search                   |     ✕     |    ✕   |   ✕  |    ✔    |
+| Sane template engine     |     ✔     |    ✔   |  ✕✕✕ |    ✔    |
+| Themes                   |     ✔     |    ✕   |   ✔  |    ✔    |
+| Shortcodes               |     ✔     |    ✕   |   ✔  |    ✔    |
+| Internal links           |     ✔     |    ✕   |   ✔  |    ✔    |
+| Table of contents        |     ✔     |    ✕   |   ✔  |    ✔    |
+| Automatic header anchors |     ✔     |    ✕   |   ✔  |    ✔    |
+| Aliases                  |     ✔     |    ✕   |   ✔  |    ✔    |
+| Pagination               |     ✔     |    ✕   |   ✔  |    ✔    |
```

Comment:
Should there be a row for listing supported text formats like markdown?

What about data files and supported formats?

Reply:
> Should there be a row for listing supported text formats like markdown?

Good point I'll add that.

> What about data files and supported formats?

Do you mean https://gohugo.io/templates/data-templates/ ? I've never used that, I'll need to read it first.

Reply:
> Do you mean https://gohugo.io/templates/data-templates/ ? I've never used that, I'll need to read it first.

Yup, [jekyll](https://jekyllrb.com/docs/datafiles/) has it too.

I've not used it yet either, so I'm too familiar with use cases for it.  I just saw it used elsewhere, saw it was trivial to implement with serde, and implemented it :).

Reply:
I have to say, even the examples in the Hugo and Jekyll docs make it look useless. I can see the `getJSON` or `getCSV` from Hugo being useful but it's orthogonal to the concept of data files. I'll add it for completeness though